packages/Search/examples/distributed_tree_driver/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# ##---------------------------------------------------------------------------##
+# ## EXAMPLES
+# ##---------------------------------------------------------------------------##
+
+TRIBITS_ADD_EXECUTABLE(
+  distributed_tree
+  SOURCES distributed_tree_driver.cpp
+  )
+IF(Kokkos_ENABLE_Serial)
+  TRIBITS_ADD_TEST(
+    distributed_tree
+    POSTFIX_AND_ARGS_0 serial --node=serial
+    COMM serial mpi
+    NUM_MPI_PROCS 4
+    FAIL_REGULAR_EXPRESSION "data race;leak;runtime error"
+    )
+ENDIF()
+IF(Kokkos_ENABLE_Cuda)
+  TRIBITS_ADD_TEST(
+    distributed_tree
+    POSTFIX_AND_ARGS_0 cuda --node=cuda
+    COMM serial mpi
+    NUM_MPI_PROCS 4
```

Comment:
Maybe 2 is enough.

Reply:
So, 1 is not enough but 4 is too many...

Reply:
Admittedly, I haven't looked how long it takes to run these new tests.